### DISCUSSION TITLE
How to exclude forks from `repository_random` option?

### DISCUSSION THREAD

**Discussion by Andre601** [lowlighter/metrics#1198] [Q&A] [answered]
With the `repository_random` option (Thanks for implementing it and helping me making it work), the listed repositories may be forks I have under my user account.

How could I exclude them?

**Comment by lowlighter** [comment, marked as answer]
Currently it's not supported yet, but if I'm not mistaken it shouldn't be too hard to do so (just pacthing the GraphQL queries)

I think all categories that were added in previous version should respect the existing [`repositories_forks`](https://github.com/lowlighter/metrics/blob/master/source/plugins/base/README.md#repositories_forks) option

**Reply by lowlighter** [reply]
This was added a few months ago
